omni-relayer/src/config.rs

```diff
@@ -21,11 +31,46 @@ fn derive_evm_address_from_private_key() -> OmniAddress {
     )
 }
 
+fn derive_base_address_from_private_key() -> OmniAddress {
+    let private_key = get_evm_private_key(ChainKind::Base);
+
+    let secret_key = SigningKey::from_slice(
+        &hex::decode(private_key).expect("Failed to decode `BASE_PRIVATE_KEY` env variable"),
+    )
+    .expect("Failed to create a `SecretKey` from the provided private key");
+
+    let signer = LocalSigner::from_signing_key(secret_key);
+
+    OmniAddress::Base(
+        H160::try_from_slice(signer.address().as_slice())
+            .expect("Failed to create `OmniAddress` from the derived public key"),
+    )
+}
+
+fn derive_arb_address_from_private_key() -> OmniAddress {
```

Comment:
These functions look very similar. Maybe it would be better to use something like `derive_evm_address_from_private_key(chain_kind: ChainKind)`?

Reply:
I tried doing this. Unfortunately, function must not have any arguments in order to be used in a serde macro

Reply:
Since we don't need a relayer on evm anymore (we needed relayer only for native fee claiming), then this logic is now useless and I completely removed it